usr/lib/common/new_host.c

```diff
@@ -3638,7 +3638,7 @@ CK_RV SC_GenerateKeyPair(STDLL_TokData_t *tokdata,
     }
 
     TRACE_DEBUG("Private Template:\n");
-    attr = pPublicKeyTemplate;
+    attr = pPrivateKeyTemplate;
```

Comment:
Please check if this has to be corrected in the other copies of new_host.c as well (ep11 token and icsf token).

Reply:
needs also to get fixed in the other new_host.c files ...
I'll do that - maybe split these fixes into a new commit.